docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+| 7.1.6.1        | POC                                                                                                                                                                                                                                                                                                                                                       | Creation of a solution design for the “Hausanschluss” Bundle.                                                                                                                                                                                                                                                                                                                                                                                                                                                    | The solution design for the ""Hausanschluss"" Bundle ispublished in the repository: https://github.com/eclipse-tractusx/tractus-x-umbrella." |
+| 7.1.6.2        | Deployment Strategy                                                                                                                                                                                                                                                                                                                                       | Scalability of Enablement Services                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | "Developers should be able to deploy an arbitrary number of Enablement Services.                                                             |
+
+# 3 AS-IS Analysis
+
+## 3.1 People
+
+### 3.1.1 Target Group Analysis
+
+See "4.1.1 To-be Target Group Analysis" below. No deviations between As-is and To-be target groups have been identified.
+
+## 3.2 Process
+
+### 3.2.1 Use Case View (high-level)
+
+Umbrella Chart is intended for the following Use Cases:
+
+| Actor             | Use case                                                                                             |
+|-------------------|------------------------------------------------------------------------------------------------------|
+| Product Developer | Sandbox Environments: Create local environments to evaluate and experiment with Catena-X components. |
+| Product Developer | Development: Provide a unified setup for contributors and developers working on Tractus-X projects.  |
+| Product Tester    | Testing: Run end-to-end tests for Catena-X services.                                                 |
```

Comment:
Product Manager is one more important Target Group we should add. Evaluating the data space .... This could be added later....

Reply:
added the PM to use cases